### PR TITLE
[script.grab.fanart] 0.16.0

### PR DESCRIPTION
**Pull request by robweber**
### Description

No major changes from the version that's in the Leia repo. Just updated a bit for some Matrix changes. Biggest change is moving `service.py` file and creating a `launcher.py` file as guidance is to have less than 15 lines in the launcher script. 

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by robweber** [comment]
I've noticed this seems like it's stuck or something - the Addon Checker has been waiting for a while.

File: script.grab.fanart/.gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+*.pyo
```
Comment (enen92) [inline script.grab.fanart/.gitignore:1]: Drop from the submission

File: script.grab.fanart/.travis.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+dist: xenial
```
Comment (enen92) [inline script.grab.fanart/.travis.yml:1]: Remove from submission

**Review by enen92 — CHANGES_REQUESTED** [review]
See comments

File: script.grab.fanart/.travis.yml (hunk 2 above)
Comment (robweber) [reply script.grab.fanart/.travis.yml:1]: Sorry about that, I thought the addon submitter addon would remove these (I think it does for my other addons). I'll get to work on it.

**Comment by enen92** [comment]
You need to add them to .gitattributes with export-ignore

**Comment by robweber** [comment]
Should be all ready now

**Review by enen92 — APPROVED** [review]

**Pull request merged**